openpgp/openpgp.d.ts

```diff
@@ -377,7 +384,18 @@ declare module openpgp.key {
      */
     interface Key {
         armor(): String,
-        decrypt(passphrase: String): Boolean,
+        decrypt(passphrase: string): boolean;
+        getExpirationTime(): Date;
+        getKeyIds(): Array<any>;
```

Comment:
According to official doc, it should return an array of Keyid :
 getKeyIds(): Array<Keyid>;